tuned/logs.py

```diff
@@ -104,8 +104,8 @@ def switch_to_console(self):
 		self.remove_all_handlers()
 		self.addHandler(self._console_handler)
 
-	def switch_to_file(self, filename = consts.LOG_FILE):
-		self._setup_file_handler(filename)
+	def switch_to_file(self, filename = consts.LOG_FILE, maxBytes = consts.LOG_FILE_MAXBYTES, backupCount = consts.LOG_FILE_COUNT):
```

Comment:
The line is too long, split it up. Let's try to make the lines 80 characters long max, where reasonable.